carp.deployments.core/src/commonTest/kotlin/dk/cachet/carp/deployments/application/ParticipationServiceTest.kt

```diff
@@ -76,6 +76,42 @@ interface ParticipationServiceTest
         assertEquals( setOf( expectedAssignedDevice ), retrievedInvitation.assignedDevices )
     }
 
+    @Test
+    fun getActiveParticipationInvitations_with_multiple_participation_succeeds() = runTest {
+        val (participationService, deploymentService, accountService) = createSUT()
+        val protocol = createSinglePrimaryDeviceProtocol()
+        val identities = listOf(
+            AccountIdentity.fromEmailAddress("father@test.com"),
+            AccountIdentity.fromEmailAddress("mother@test.com")
+        )
+        val invitation = StudyInvitation("Test study", "description", "Custom data")
+        val participantInvitations = identities.map { identity ->
+            ParticipantInvitation(
+                participantId = UUID.randomUUID(),
+                AssignedTo.All,
```

Comment:
This means that both accounts will be assigned to _all_ roles (both "father" and "mother"), which while possible, is misleading and makes this example less sensical.